platform/src/main/java/platform/pageobjects/Footer/FooterPage_PT90_106.java

```diff
@@ -0,0 +1,713 @@
+	public WebElement qnALink;
+
+	@FindBy(xpath = "//ul[@class='footer__link']//a[@href='/testing']")
+	public WebElement footerExaminationWindowLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='https://studygear.evidus.com/campaign/cp_scbt202109/']")
+	public WebElement footerScbtLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='https://casec.evidus.com/about/']")
+	public WebElement footerCasecLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/testing/achievement-test_pf']")
+	public WebElement footerCommontestZKai;
+
+	@FindBy(xpath = "//ul[@class='footer__link']//a[@href='/learning']")
+	public WebElement footerLearningWindowLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/studygear-eiken']")
+	public WebElement footerStagiaEikenLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/studygear-kanken']")
+	public WebElement footerStagiaKankenLink;
+
+	@FindBy(xpath = "//ul[@class='footer__link']//a[@href='/media']")
+	public WebElement informationWindowLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='https://stg.jyuku-pita.evidus.com/']")
+	public WebElement cramSchoolPitaLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='https://juken.ei-navi.jp/']")
+	public WebElement stagiaUniversityExamLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='https://consult.evidus.com/']")
+	public WebElement educationCostConsulatationSupportLink;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/mypage-account-security']")
+	public WebElement accountSecurity;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/mypage-member-status']")
+	public WebElement memberStatus;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/mypage-member-service']")
+	public WebElement servicesUsing;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/order-history']")
+	public WebElement orderHistory;
+
+	@FindBy(xpath = "//a[@href='/withdraw-confirm']")
+	public WebElement withdraw;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/logout']")
+	public WebElement logout;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/login']")
+	public WebElement footerLogin;
+
+	@FindBy(xpath = "//li[@class='sub-menu__item']//a[@href='/register']")
+	public WebElement footerRegister;
+
+	@FindBy(xpath = "//a[@href='https://support.evidus.com/']")
+	public WebElement footerAccountQnA;
+
+	@FindBy(xpath = "//div[@class='ep-auth__footer']/ul[1]/li[2]//a[1]")
+	public WebElement footerAccountTerms;
+
+	@FindBy(xpath = "//a[@href='https://www.jiem.co.jp/privacy/']")
+	public WebElement footerAccountPrivacy;
+
+	@FindBy(xpath = "//div[@class='ep-auth__footer']/ul[1]/li[4]//a[1]")
+	public WebElement footerAccountTax;
+
+	@FindBy(xpath = "//h3[contains(text(),'「スタギア」利用規約')]")
+	public WebElement footerTermsforAssert;
+
+	@FindBy(xpath = "//img[@alt='個人情報保護方針']")
```

Comment:
PLz replace if possible. img tag is not advisable

Reply:
Updated Webelement name
Following Webelement is used for visibility.

Reply:
Updated xpaths as requested.